packages/connect-core/src/grpc-web-create-request-header.ts

```diff
@@ -16,14 +16,16 @@
  * Creates headers for a gRPC-web request.
  */
 export function grpcWebCreateRequestHeader(
+  useBinaryFormat: boolean,
```

Comment:
:+1: Now that we have added support for JSON with gRPC-web for connect-node, we should add this option to connect-web as well.

Reply:
Added in https://github.com/bufbuild/connect-web/pull/334